VespaGAS.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
What do you mean by "having a `==` function is not the same as conforming to Equatable"? 

The provided `==` function cover both Oil and Gasoline, why should I provided another custom implementation of `==` if they are already covered by the generic one?